velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1289,6 +1289,28 @@ TEST_F(StringFunctionsTest, HmacSha512) {
   EXPECT_EQ(std::nullopt, hmacSha512(std::nullopt, "velox"));
 }
 
+TEST_F(StringFunctionsTest, SpookyHashV2_32) {
+  const auto spookyHashV2_32 = [&](std::optional<std::string> arg) {
+    return evaluateOnce<std::string, std::string>(
+        "spooky_hash_v2_32(c0)", {arg}, {VARBINARY()});
+  };
+  // Use the same expected value from TestVarbinaryFunctions of presto java
```

Comment:
Please add some more test cases :
i) Varchar values with special characters : #,@,$,%,^~ etc
ii) Values with numeric characters
iii) Varchar values that are > 12 characters in length. This causes the Varchar to use out of line storage.